tests/unittest/_torch/modeling/test_modeling_deepseekv4.py

```
@@ -0,0 +1,869 @@
+    config.sparse_attention_config = sparse_attn_config
+
+    device = torch.device("cuda")
+    # with default_dtype(config.dtype):
+    model_config = ModelConfig(
+        pretrained_config=config, sparse_attention_config=sparse_attn_config, attn_backend="TRTLLM"
+    )
+    model = DeepseekV4ForCausalLM(model_config).to(device)
+    assert not model.model.layers[0].fusion_config.POST_MOE_FUSION
+
+    context_sequence_length = [3, 2, 5]
+    sequence_length = context_sequence_length + [1, 1]
```

Ruff (0.15.18)

[warning] Consider `[*context_sequence_length, 1, 1]` instead of concatenation

Replace with `[*context_sequence_length, 1, 1]`

(RUF005)